suite/tests/linux/clone.c

```diff
@@ -221,7 +277,7 @@ create_thread(void (*fcn)(void), void **stack, bool share_sighand, bool clone_vm
     return newpid;
 }
 
-/* glibc does not provide a wrapper for clone3 yet. This makes it difficult
+/* drlibc does not provide a wrapper for clone3 yet. This makes it difficult
```

Comment:
Probably this should say "Neither glibc nor drlibc provides..."

Reply:
Done